yarn.lock

```diff
@@ -4474,6 +4474,11 @@ monaco-editor-core@^0.17.0:
   resolved "https://registry.yarnpkg.com/monaco-editor-core/-/monaco-editor-core-0.17.0.tgz#8d2b7f7e02ac68f68eecc881d7a26457cbc09b49"
```

Comment:
`monaco-editor-core@^0.17.0` should be removed, could you check why it is pulled, i.e. with `yarn why monaco-editor-core@0.17.0`